src/librustc/ty/context.rs

```diff
@@ -1089,6 +1094,51 @@ impl<'tcx> TyCtxt<'tcx> {
         &self.hir_map
     }
 
+    #[inline]
+    pub fn get_interp_user_fn(&self) -> DefId {
```

Comment:
A doc comment re. where this is used, what is expected there, and what function it plays there would be good.